crates/notary/server/src/settings.rs

```diff
@@ -0,0 +1,77 @@
+use config::{Config, ConfigError, Environment, File};
+use std::path::Path;
+use tracing::{info, warn};
+use crate::{CliFields, NotaryServerProperties};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+    #[serde(flatten)]
+    pub config: NotaryServerProperties,
+}
+
+impl Settings {
+    pub fn new(cli_fields: &CliFields) -> Result<Self, ConfigError> {
+        let mut builder = Config::builder();
+
+        // Add default values
+        builder = builder

```

Comment:
can we change these to use the config file as the default instead, so that we have these default values only in one location